src/components/create-lazy-component.js

```diff
@@ -0,0 +1,23 @@
+import universal from 'react-universal-component';
```

Comment:
@rsumilang change filename to match function name `createLazyComponent.js` vs the dashed equivalent.  convention we've landed on is that components are named as if they are web components, e.g. `date-picker` and has its own directory (whether it's screen, container, dumb component).  everything else is named to match whatever it is.